front-end/src/components/modal/create/VideoSelect.js

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+import '../../../styles/modalStyles.css';
+import '../../../styles/videoSelectStyles.css';
+import { AiFillFolder } from 'react-icons/ai';
+import { filterFiles } from '../../../lib/videoUtils';
+
+export const VideoSelect = ({ movePage, selectedVideo, setSelectedVideo, files, fileTimespans, videoTimespans }) => {
+
+    const [filteredFiles, setFilteredFiles] = useState([]);
+
+    useEffect(() => {
+        setFilteredFiles(filterFiles(selectedVideo, files, fileTimespans))
+    }, [selectedVideo]);
+
+    //render the modal JSX in the portal div.
```

Comment:
This is entirely insignificant but comment formatting perhaps :>

`// Render the modal JSX in the portal div.`

Reply:
removed the comment